common/test/acceptance/pages/studio/edit_tabs.py

```diff
@@ -14,3 +16,86 @@ class PagesPage(CoursePage):
 
     def is_browser_on_page(self):
         return self.q(css='body.view-static-pages').present
+
+    def is_static_page_present(self):
+        """
+        Checks for static tab's presence
+        Returns: True or False
+        """
+        return self.q(css='.wrapper.wrapper-component-action-header').present
+
+    def add_static_page(self):
+        """
+        Adds a static page
+        """
+        total_tabs = len(self.q(css='.course-nav-list>li'))
+        click_css(self, '.add-pages .new-tab', require_notification=False)
+        self.wait_for(
+            lambda: len(self.q(css='.course-nav-list>li')) == total_tabs + 1,
+            description="Static tab is added",
+            timeout=3
+        )
+        self.wait_for_element_visibility('.tab-list :nth-child({})'.format(total_tabs), 'Static tab is visible')
+        self.wait_for_ajax()
+
+    def delete_static_tab(self):
+        """
+        Deletes a static page
+        """
+        click_css(self, '.btn-default.delete-button.action-button', require_notification=False)
+        confirm_prompt(self)
+
+    def click_edit_static_page(self):
+        """
+        Clicks on edit button to open up the xblock modal
+        """
+        self.q(css='.edit-button').first.click()
+        EmptyPromise(
+            lambda: self.q(css='.xblock-studio_view').present,
+            'Wait for the Studio editor to be present'
+        ).fulfill()
+
+    def drag_and_drop_first_static_page_to_last(self):
+        """
+        Drags and drops the first the static page to the last
+        """
+        draggable_elements = self.q(css='.component .drag-handle').results
+        source_element = draggable_elements[0]
+        target_element = draggable_elements[-1]
+        action = ActionChains(self.browser)
+        action.click_and_hold(source_element)
+        action.move_to_element_with_offset(target_element, 0, 50)
```

Comment:
try using action.drag_and_drop(source, target).perform() and see if it works

Reply:
It did not work, so I have replaced the code piece.